src/nl/hannahsten/texifyidea/psi/LatexParameterTextUtil.kt

```diff
@@ -60,6 +69,7 @@ fun getNameIdentifier(element: LatexParameterText): PsiElement? {
         !CommandMagic.labelDefinitionsWithoutCustomCommands.contains(name) &&
         !CommandMagic.bibliographyReference.contains(name) &&
         !CommandMagic.labelAsParameter.contains(name) &&
+        !CommandMagic.glossaryEntry.contains(name) &&
```

Comment:
> the getNameIdentifier in LatexParameterTextUtil must return the element even if the selected parameter is not the label/identifier parameter. 

Why is that? I would think you could just return null if the element in question is not an identifier?
Or did you mean that this isn't yet implemented for other commands so you didn't bother to do it either, in which case I agree, because I think this was just for the specific inspection mentioned.

Reply:
What I mean is that each and every required parameter (or key/val) is a `parameter_text` which in turn is a `PsiNamedElement`. I actually tried to implement something like "for glossary entries return the element, i.e., the parameter, only if it is the first parameter", but then trying to rename any other parameter leads to the documented exception because the inplace renamer expectes an identifier whenever the element is a `PsiNamedElement`.

Reply:
> I'll make a note of it.

Let me know if I can help with this topic. I think a few things could be solved more elegantly with a conditional parser, but it might have long reaching consequences. I played a bit with an external parser rule while waiting for your answer, but couldn't extract the parent command token when parsing a parameter completely reliably yet.